README.md

```diff
@@ -1,11 +1,224 @@
+| --- | --- |
+| 111122223333 | `Program Part A / 123456` |
+| 222233334444 | `Program Part B / 123456` |
+| 333344445555 | `Other Program / 654321` |
+
+
+#### Example Output
+A JSON string to be passed to a Rules property of an AWS::CE::CostCategory resource:
+```json
+[
+    {
+        "Type": "REGULAR",
+        "Value": "123456 Program Part A",
+        "Rule": {
+            "Tags": {
+                "Key": "CostCenterOther",
+                "Values": [ "123456" ],
+                "MatchOptions": [ "ENDS_WITH" ]
+            }
+        }
+    },
+    {
+        "Type": "REGULAR",
+        "Value": "123456 Program Part A",
+        "Rule": {
+            "Tags": {
+                "Key": "CostCenter",
+                "Values": [ "123456" ],
+                "MatchOptions": [ "ENDS_WITH" ]
+            }
+        }
+    },
+    {
+        "Type": "REGULAR",
+        "Value": "123456 Program Part A",
+        "Rule": {
+            "Dimensions": {
+                "Key": "LINKED_ACCOUNT",
+                "Values": [ "111122223333", "222233334444" ],
+                "MatchOptions": [ "EQUALS" ]
+            }
+        }
+    },
+    {
+        "Type": "REGULAR",
+        "Value": "654321 Other Program",
+        "Rule": {
+            "Tags": {
+                "Key": "CostCenterOther",
+                "Values": [ "123456" ],
+                "MatchOptions": [ "ENDS_WITH" ]
+            }
+        }
+    },
+    {
+        "Type": "REGULAR",
+        "Value": "654321 Other Program",
+        "Rule": {
+            "Tags": {
+                "Key": "CostCenter",
+                "Values": [ "123456" ],
+                "MatchOptions": [ "ENDS_WITH" ]
+            }
+        }
+    },
+    {
+        "Type": "REGULAR",
+        "Value": "654321 Other Program",
+        "Rule": {
+            "Dimensions": {
+                "Key": "LINKED_ACCOUNT",
+                "Values": [ "333344445555" ]
+                "MatchOptions": [ "EQUALS" ]
+            }
+        }
+    },
+    {
+        "Type": "INHERITED_VALUE",
+        "InheritedValue": {
+            "DimensionName": "TAG",
+            "DimensionKey": "CostCenterOther"
+        }
+    },
+    {
+        "Type": "INHERITED_VALUE",
+        "InheritedValue": {
+            "DimensionName": "TAG",
+            "DimensionKey": "CostCenter"
+        }
+    }
+]
+```
+
+This string can be passed to the [`Rules` property for `AWS::CE::CostCategory`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ce-costcategory.html#cfn-ce-costcategory-rules).
+
+### Example Usage (sceptre)
+
+To use the output of this lambda in [sceptre](https://github.com/Sceptre/sceptre), use `wget` in a [`!rcmd` resolver](https://docs.sceptre-project.org/dev/docs/resolvers.html#file)
+
+`config/dev/ce.yaml`
+```yaml
+template:
+  path: categories.yaml
+parameters:
+  CostCategoryRules: !rcmd wget -qO- https://lambda-finops-cost-rules.execute-api.amazonaws.com/rules
```

Comment:
i think `rcmd` will work but i think using the [sceptre-request-resolver](https://github.com/Sceptre/sceptre-request-resolver/pull/1) would be better.

Reply:
I agree, using the new resolver would be better